src/services/developer/developer.svc.coffee

```diff
@@ -27,14 +27,7 @@ angular
 
```

Comment:
Have you tested that the widget stubbing works with the dev toolkit? Can test really quickly with the toolkit settings page. Looks good just wondering if you've tried it